platform/o.n.core/src/org/netbeans/core/NbLifecycleManager.java

```diff
@@ -67,9 +67,6 @@ public static void advancePolicy() {
         if (!Boolean.getBoolean("TopSecurityManager.disable")) {
             // set security manager
             TopSecurityManager.install();
-            if (CLIOptions.isGui()) {
-                TopSecurityManager.makeSwingUseSpecialClipboard(Lookup.getDefault().lookup(ExClipboard.class));
```

Comment:
What was this supposed to do ?

Reply:
I believe this was part of a workaround for JDK-4818143. That bug is marked as fixed in JDK 6. So, it seems that keeping the workaround is probably not useful anymore (and, in case the problem would not be actually fixed, this is a way to find out).

Reply:
Doesn't that also stop the system clipboard being an instance of ExClipboard?  If so, seem a bunch of places that expect it to be one.